packages/playground/blueprints/src/lib/steps/activate-theme.ts

```diff
@@ -18,18 +18,13 @@ export const activateTheme: StepHandler<ActivateThemeStep> = async (
 	progress
 ) => {
 	progress?.tracker.setCaption(`Activating ${themeFolderName}`);
-	const themesPage = asDOM(
-		await playground.request({
-			url: '/wp-admin/themes.php',
-		})
-	);
-
-	const link = themesPage.querySelector(
-		`a[href*="action=activate&stylesheet=${themeFolderName}"]`
-	)! as HTMLAnchorElement;
-	const href = link.attributes.getNamedItem('href')!.value;
-
-	await playground.request({
-		url: href,
-	});
+	const wpLoadPath = `${playground.documentRoot}/wp-load.php`;
+	if (playground.fileExists(wpLoadPath)) {
+		await playground.run({
+			code: `<?php
+      include_once( '${wpLoadPath}' );
+      switch_theme( '${themeFolderName}' );
+      `,
+		});
```

Comment:
Lovely!

Reply:
Can the same be done for plugins?

Reply:
But can you activate plugins with PHP? Because if you can then we don't need jsdom at all.

Reply:
@adamziel , done.
It was hard to make it work. Including `/wp-admin/plugins.php` was redirecting to `wp-login.php`.

Reply:
Let's stick to the plugin slug and auto-prefix it with the absolute path of wp-content/plugins – plugins should never be installed in another location.

Reply:
> It was hard to make it work. Including /wp-admin/plugins.php was redirecting to wp-login.php

Thank you for taking the time to figure it out ❤️